index.js

```diff
@@ -12,6 +12,7 @@ const STORY_STATES = {
 const LABEL_STORY_MAP = {
   'needs more work': STORY_STATES.IN_DEVELOPMENT,
   'ready for review': STORY_STATES.READY_FOR_REVIEW,
+  'ready to merge': STORY_STATES.COMPLETED,
```

Comment:
We should not do this; merging the PR already does mark the story completed so this just prematurely marks it as completed.

Reply:
@ibrahima Ohhh, didn't know that. Removed that line.